src/main/resources/templates/fragments/topNav.html

```diff
@@ -0,0 +1,23 @@
+<div th:fragment="navbar">
```

Comment:
I took the navBar out of the html files and created a fragment for it. Every page will have the same navBar so it will save time and effort by making it a component. The navBar also had an id of "one". Unless for some reason you need to programatically access the entire navBar it does not need an id. And if you do need an id it needs a better id such as "navBar"